File: JavaScript/7kyu/SumOfLowest.js
function sumTwoSmallestNumbers(numbers) {
  return numbers
    .sort((a, b) => b - a)
    .slice(-2)
    .reduce((a, b) => a + b, 0);
}

function sumTwoSmallestNumbers_2(numbers) {
  var [a, b] = numbers.sort((a, b) => a - b);
  return a + b;
}

var result = sumTwoSmallestNumbers_2([15, 28, 4, 2, 43]);
console.log(result);

File: CSharp/7kyu/SumOfLowest.cs
using System;
using System.Linq;

namespace CSharp._7Kyu
{
    public static partial class Kata7
    {
        public static int sumTwoSmallestNumbers(int[] numbers)
        {
            int min_1 = numbers.Min();
            int index = Array.IndexOf(numbers, min_1);
            int min_2 = numbers.Where((v, i) => i != index).Min();

            return min_1 + min_2;
        }

        public static int sumTwoSmallestNumbers_2(int[] numbers)
        {
            return numbers.OrderBy(i => i).Take(2).Sum();
        }
    }
}